geode-core/src/integrationTest/java/org/apache/geode/internal/cache/OfflineSnapshotJUnitTest.java

```diff
@@ -65,6 +65,16 @@ public void testExport() throws Exception {
     }
   }
 
+  private void removeAndInvalidateKeys(Region<Integer, MyObject> region,
+      Map<Integer, MyObject> expected) {
+    int removed = 5;
+    int invalidated = 7;
+    region.destroy(removed);
+    region.invalidate(invalidated);
+    expected.remove(removed);
+    expected.put(7, null);
```

Comment:
Is `null` different than `Token.isInvalid`?  I'm wondering if we import a null value that will have the same behavior as if the Token is used.

Reply:
Based on SnapshotPacket `convertToBytes` I think null is correct.

Reply:
During snapshot import, null value is converted to Token.INVALID to use the putAll operation.